commit a7de66ffb0e6e2b349badff9672fc1609079de64
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-06-12T14:31:39+01:00 (4 months ago)
    
    MANTA-4313 sdc-manta transitive dependencies should not use git+ssh:// URIs in package.json files

diff --git a/cmd/manta-shardadm.js b/cmd/manta-shardadm.js
index efd0f2b..a3fb50b 100755
--- a/cmd/manta-shardadm.js
+++ b/cmd/manta-shardadm.js
@@ -6,7 +6,7 @@
  */
 
 /*
- * Copyright (c) 2014, Joyent, Inc.
+ * Copyright 2019 Joyent, Inc.
  */
 
 /*
@@ -66,7 +66,8 @@ Shardadm.prototype.init = function (opts, args, cb) {
 	this.client = new sdc.SAPI({
 	    url: config.sapi.url,
 	    log: this.log,
-	    agent: false
+		agent: false,
+		version: '~2'
 	});
 
 	Cmdln.prototype.init.apply(this, arguments);
diff --git a/lib/common.js b/lib/common.js
index 479dfbb..4c92cc5 100644
--- a/lib/common.js
+++ b/lib/common.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2019, Joyent, Inc.
+ * Copyright 2019 Joyent, Inc.
  */
 
 /*
@@ -17,6 +17,7 @@ var fs = require('fs');
 var path = require('path');
 var restifyClients = require('restify-clients');
 var sdc = require('sdc-clients');
+var ufds = require('ufds');
 var vasync = require('vasync');
 var VError = require('verror').VError;
 
@@ -140,7 +141,8 @@ function initSdcClients(cb) {
 		self.SAPI = new sdc.SAPI({
 			log: self.log,
 			url: config.sapi.url,
-			agent: false
+			agent: false,
+			version: '~2'
 		});
 
 		self.AMON = new sdc.Amon({
@@ -155,7 +157,7 @@ function initSdcClients(cb) {
 			agent: false
 		});
 
-		self.UFDS = new sdc.UFDS({
+		self.UFDS = new ufds({
 			log: self.log,
 			url: config.ufds.url,
 			bindDN: config.ufds.bindDN,
diff --git a/package.json b/package.json
index 078e56e..6956625 100644
--- a/package.json
+++ b/package.json
@@ -15,7 +15,7 @@
         "fash": "2.5.0",
         "forkexec": "0.1.0",
         "hogan.js": "2.0.0",
-        "imgapi-cli": "git+https://github.com/joyent/sdc-imgapi-cli.git#65bba66818",
+        "imgapi-cli": "git+https://github.com/joyent/sdc-imgapi-cli.git#5e2b2ae757",
         "jsprim": "^1.4.0",
         "js-yaml": "3.8.2",
         "node-uuid": "1.4.0",
@@ -25,10 +25,11 @@
         "progbar": "1.1.1",
         "readable-stream": "1.0.26",
         "restify-clients": "1.5.0",
-        "sdc-clients": "git+https://github.com/joyent/node-sdc-clients.git#42ea07d3ca",
+        "sdc-clients": "13.0.1",
         "sprintf-js": "0.0.7",
         "tab": "0.1.0",
         "triton-netconfig": "1.4.0",
+        "ufds": "1.6.0",
         "urclient": "1.0.0",
         "vasync": "1.6.3",
         "verror": "1.10.0",
